nncf/tensorflow/tf_internals.py

```diff
@@ -18,7 +18,7 @@
 
 from nncf.tensorflow import tensorflow_version
 
-if version.parse(tensorflow_version) < version.parse("2.13"):
+if tensorflow_version < version.parse("2.13"):
```

Comment:
Why can't we fix `2.13` somewhere as a variable instead of spreading hard-coded versions everywhere?

Reply:
What are you suggesting?